Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM python:slim-bullseye
+
+RUN useradd -m agr-mvs
+RUN apt-get update && apt-get install -y curl
+USER agr-mvs
+ENV POETRY_HOME="/home/agr-mvs/.local/poetry"
+ENV PATH="$POETRY_HOME/bin:$PATH" \
+    HOME="/home/agr-mvs"
+
+RUN curl -sSL https://install.python-poetry.org | python3 - && pip install poethepoet
```

Comment:
¿Por qué usas python3 y no simplemente python? No hay otra versión ahora mismo que tenga soporte.

Reply:
Aparte que si estás usando `pip` para instalar poethepoet, ¿qué problema hay para que lo uses también para instalar `poetry`? Y te ahorras instalar nada para bajártelo.
De veras, que tenéis que pensar realmente lo que estáis usando y  por qué.